sdk-setup/src/mb2

```diff
@@ -58,8 +58,8 @@ COMMANDS OVERVIEW
 
    build-init  Initialize build directory
    build-requires  Update build time dependencies
-   prepare  Prepare sources using the recipe from spec file
-   apply  Apply patches listed in the spec file
+   prepare  Prepare sources using the recipe from RPM SPEC file
+   apply  Apply patches listed in the SPEC file
```

Comment:
In this kind of listing I would prefer to say RPM with every occurrence of SPEC.